src/main/java/com/glodblock/github/client/gui/GuiLevelMaintainer.java

```diff
@@ -411,6 +418,10 @@ public void setEnable(boolean enable) {
 
         private boolean send(Widget widget) {
             if (((SlotFluidConvertingFake) cont.inventorySlots.get(widget.idx)).getHasStack()) {
+                if (isGTNHLibLoaded) {
+                    long value = (long) MathExpressionParser.parse(widget.textField.getText());
+                    widget.textField.setText(String.valueOf(value));
+                }
                 if (!widget.textField.getText().isEmpty() && widget.textField.getText().matches("^[0-9]+")) {
```

Comment:
`numberPattern` can be used here as well.

Reply:
👍